lang/python/python-uci/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2018-2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-uci
+PKG_VERSION:=0.8.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=pyuci
+PKG_HASH:=9287fe41b427dc5c167592d429be48c1e6cfe276225681b1bdefddfe90d7e941
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-uci
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python UCI bindings
+  URL:=https://gitlab.nic.cz/turris/pyuci/
+  DEPENDS:=+python3-light +libuci
+  VARIANT:=python3
```

Comment:
Sorry! I missed this thing. This is no longer necessary. This is used in OpenWrt 19.07 version, which has two version of Pythons and in OpenWrt master, there is only one, so this can be removed.

```suggestion
```